spring-integration-core/src/main/java/org/springframework/integration/dsl/IntegrationFlowDefinition.java

```diff
@@ -213,6 +213,18 @@ public B channel(MessageChannel messageChannel) {
 	}
 
 	/**
+	 * Populate a {@link MessageChannel} instance
+	 * at the current {@link IntegrationFlow} chain position using the {@link Channels}
+	 * factory fluent API.
+	 * @param channels the {@link Function} to use.
+	 * @return the current {@link IntegrationFlowDefinition}.
+	 */
+	public B channel(Function<Channels, MessageChannelSpec<?, ?>> channels) {
```

Comment:
The JMS test case uses this method - adding it back solved that - not sure what I am missing.